server/tests/article.test.js

```diff
@@ -15,24 +15,24 @@ const user = {
 };
 
 const article = {
-  title: faker.lorem.sentence(),
-  body: faker.lorem.paragraph(),
-  description: faker.lorem.sentences()
+  title: 'When you love',
+  body: 'body exercise is good and better',
+  description: 'Physical and health'
 };
 
 const noTitle = {
-  body: faker.random.words(),
-  description: faker.random.words()
+  body: 'body exercise is good and better',
+  description: 'Physical and health'
 };
 
 const noBody = {
-  title: faker.random.words(),
-  description: faker.random.words()
+  title: 'When you love',
+  description: 'Physical and health'
 };
 
 const noDescription = {
-  title: faker.random.words(),
-  body: faker.random.words()
```

Comment:
Faker library is causing irregularity on the test case, so I decided to replace it with plain text.